auth/README.md

```diff
@@ -1,38 +1,39 @@
 # Auth Package
 
-Auth package is used for request authentication and authorization for user to service
-and service to service communication. Auth is used for scenarios that require stricter
-validations, and usage restrictions. `auth.Client` provided by the package provides seamless
+Use `package auth` for request authentication and authorization for user-to-service
```

Comment:
For example:
- Line 3: `package auth`
- Line 26: `Auth package`
- Line 4: auth
- Line 6: Auth
- Line 9: The Auth package

We should be somewhat consistent, like capital A or lowercase a, backticks or no backticks? I think "package auth" is fine, but I'm open to arguments.